examples/echo/dialer.js

```diff
@@ -1,65 +1,51 @@
 'use strict'
 /* eslint-disable no-console */
 
+/*
+ * Dialer Node
+ */
+
 const PeerId = require('peer-id')
 const PeerInfo = require('peer-info')
 const Node = require('../../src')
 const multiaddr = require('multiaddr')
 const pull = require('pull-stream')
 const async = require('async')
-let idListener
 
 async.parallel([
-  (callback) => {
-    PeerId.createFromJSON(require('./peer-id-dialer'), (err, idDialer) => {
-      if (err) {
-        throw err
-      }
-      callback(null, idDialer)
-    })
-  },
-  (callback) => {
-    PeerId.createFromJSON(require('./peer-id-listener'), (err, idListener) => {
-      if (err) {
-        throw err
-      }
-      callback(null, idListener)
-    })
-  }
+  (cb) => PeerId.createFromJSON(require('./id-d'), cb),
+  (cb) => PeerId.createFromJSON(require('./id-l'), cb)
 ], (err, ids) => {
-  if (err) throw err
-  const peerDialer = new PeerInfo(ids[0])
-  peerDialer.multiaddr.add(multiaddr('/ip4/0.0.0.0/tcp/0'))
-  const nodeDialer = new Node(peerDialer)
+  if (err) { throw err }
 
-  const peerListener = new PeerInfo(ids[1])
-  idListener = ids[1]
-  peerListener.multiaddr.add(multiaddr('/ip4/127.0.0.1/tcp/10333'))
-  nodeDialer.start((err) => {
-    if (err) {
-      throw err
-    }
+  // Dialer
+  const idD = ids[0]
```

Comment:
Could we keep the longer identifier? `dialerID` is a lot easier to read than `idD`